syft/__init__.py

```diff
@@ -48,11 +48,14 @@
 from syft.grid import VirtualGrid
 
 # Import federate learning objects
-from syft.federated import Plan
```

Comment:
Could we limit this pull request to only contain the introduction of the message type?
It is not clear to me, why Plan should be in a message folder.

Reply:
The message folder will contain all messages, groups of messages, and graphs of messages. Plan was in the federated folder which implied that it's only useful for Federated Learning.

Reply:
I agree that Plan can be used for more than federated learning. But I am not sure that it should be in a message folder neither.

Reply:
a plan is a cache of messages - seemed pretty close. Other ideas for what to name the folder?

Reply:
maybe rename the folder "messaging"?

Reply:
@mari-linhares what do you think about the naming of the folder and the location of Plan?

Reply:
I agree that we should rename it eventually. Not sure if "message" is the best option but I don't have a good naming in mind neither.

Reply:
msg, cache,... something. 

It's relatively easy to change in the future with modern IDEs. But I think all are better than having it in federated.